Automattic/wp-calypso
# Fix/reader discover unbound callbacks

This removes unbound callbacks in the `DiscoverSiteAttribution` component and removes unused `DiscoverPostAttribution`.

**Testing**
1. Enable analytics logging in the dev console:  `localStorage.setItem( 'debug','calypso:analytics:*' )`
2. visit `http://calypso.localhost:3000/discover`
3. Click the title of any post in discover stream
4. On the full post view click `visit {some site}` :
![screen shot 2017-05-16 at 1 55 16 pm](https://cloud.githubusercontent.com/assets/744755/26127927/c55fdc0a-3a3f-11e7-8f53-be650b288bf6.png)

5 The event `calypso_reader_site_on_discover_card_clicked` should be logged to the console
6. Click back to the full post view
7. Click the follow/unfollow button for the source post:
![screen shot 2017-05-16 at 1 55 16 pm](https://cloud.githubusercontent.com/assets/744755/26127937/cf768734-3a3f-11e7-8964-12431edf666b.png)
8. The events `calypso_reader_site_followed` or `calypso_reader_site_unfollowed` should fire depending on the following state of the source post

Fix for : #14133 and #14132
